src/platform/webos/DnssdImpl.h

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
Can we re-use linux-based impl files and headers instead of copying?

Reply:
@andy31415 
Yes. Let me remove this and re-use the one from Linux.
Thank you for your review.